domain/src/test/java/org/oppia/domain/question/QuestionTrainingControllerTest.kt

```diff
@@ -156,12 +160,25 @@ class QuestionTrainingControllerTest {
   // TODO(#89): Move this to a common test application component.
   @Module
   class TestModule {
+    @Mock
+    lateinit var questionTrainingConstantsProvider: QuestionTrainingConstantsProvider
+
     @Provides
     @Singleton
     fun provideContext(application: Application): Context {
       return application
     }
 
+    @Provides
```

Comment:
This is old, you can pull the changes from develop once again to update this!

Reply:
Done!